website/src/pages/PrivacyPolicy/style.ts

```diff
@@ -8,7 +8,7 @@ const useStyles = makeStyles((theme) => ({
         },
     },
     pageSpacing: {
```

Comment:
is `pageSpacing` used as a container? if thats the case i think its better to use max-width

Reply:
Good one! Replaced padding with width.